rfcs/003-randomness.md

```diff
@@ -0,0 +1,193 @@
+# RFC: Randomness
+
+Author: José Duarte — @jmg-duarte
+Date: 6/12/24
+
+## Abstract
+
+This document covers existing randomness methods in Polkadot and discusses which
+would be appropriate to the Polka Storage project.
```

Comment:
```suggestion
This document covers existing randomness methods in Polkadot and discusses which
would be appropriate to the Polka Storage project. We propose XXXX as the randomness source for Proofs.
```

Reply:
Helps a lot for potential readers to expect what they need to search for in the document  (in my opinion)